web-app/js/portal/ui/openlayers/MapOptions.js

```diff
@@ -50,6 +50,7 @@ Portal.ui.openlayers.MapOptions = Ext.extend(Object, {
             this.clickControl
         ];
 
+        this.tileManager = null;
```

Comment:
It "fixes" the problem that you mention here: https://github.com/aodn/aodn-portal/commit/63472c212a1c2cdfb4cffb6459fd7e4919cb91e2

Along with this, and having our override removed, panning + layer events works.  Like I say above, I'm not very happy with this, hopefully I haven't broken anything.